### DISCUSSION TITLE
当治安官有什么条件？

### DISCUSSION THREAD

**Discussion by sishui-buyin** [share121/inter-knot#1244] [其他]
本人17岁天天在家打火影，不知道干嘛，梦想去治安局当治安官维护一方治安，所以当治安官有什么条件？

**Comment by lllliufeng** [comment]
（一）具有新艾利都户籍；
（二）18周岁以上、35周岁以下，应届毕业硕士研究生和博士研究生（非在职）年龄可放宽到40周岁以下；
（三）拥护新艾利都宪法；
（四）具有良好的品行；
（五）具有正常履行职责的身体条件；
（六）具有符合职位要求的工作能力；
（七）具有大专以上文化程度；
（八）具备新艾利都治安局主管部门规定的拟任职位所要求的其他资格条件。

**Comment by sishui-buyin** [comment]
好吧，看来需要好好锻炼和学习了
